packages/mux-uploader-react/README.md

```diff
@@ -67,20 +67,20 @@ const MuxUploaderWithMuxUploaderDropExample = () => {
+| `url`            | `string`   | `Promise`                                                                                                                                                                                            | Either a) the authenticated URL that your file will be uploaded to or b) an async function that yields a promise that resolves to that url. Check out the [direct uploads docs](https://docs.mux.com/guides/video/upload-files-directly#1-create-an-authenticated-mux-url) for how to create one. Required. | `undefined` |
+| `id`             | `string`   | An ID that allows `<MuxUploaderDrop>` to locate `<MuxUploader>`. Required if you use `<MuxUploaderDrop>`.                                                                                            | N/A                                                                                                                                                                                                                                                                                                         |
+| `type`           | `"bar"`    | Specifies the visual type of progress bar. A radial type is in-progress.                                                                                                                             | "bar"                                                                                                                                                                                                                                                                                                       |
+| `status`         | `boolean`  | Toggles text status visibility of progress bar. The text that is displayed is a percentage by default. If you prefer just the progress bar with no text upload status, don't include this attribute. | false                                                                                                                                                                                                                                                                                                       |
+| `formatProgress` | `function` | A function that accepts numeric percent and is expected to return a string. Allows for customizing how the progress should be rendered.                                                              | A function the yields a percent progress string                                                                                                                                                                                                                                                             |
```

Comment:
polish(non-blocking): Two sentences here that are very similar. Consider revising to only need one of the two.